### PR TITLE
docs: add installation guide for Gatsby

### PR DESCRIPTION
**Pull request by moinulmoin**
image

### CODE REVIEW

File: apps/www/content/docs/installation/gatsby.mdx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+### Create project
+
+Start by creating a new Gatsby project using `Gatsby CLI`:
+
+```bash
+npx gatsby new
+```
+
+### Configure your gatsby project to use TypeScript and Tailwind CSS
```
Comment (shadcn) [inline apps/www/content/docs/installation/gatsby.mdx:16]:
```diff
- your gatsby project
+ your Gatsby project
```

File: apps/www/content/docs/installation/gatsby.mdx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+✔ Would you like to install a styling system?
+· Tailwind CSS
+✔ Would you like to install additional features with other plugins?
+· Choose whatever you want
+✔ Shall we do this? (Y/n) · Yes
+```
+
+### Edit tsconfig.json file
+
+Add the code below to the tsconfig.json file to resolve paths without error:
```
Comment (shadcn) [inline apps/www/content/docs/installation/gatsby.mdx:38]: I think we can remove the _without_error_ here.

File: apps/www/content/docs/installation/gatsby.mdx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      ]
+    }
+    // ...
+  }
+}
+```
+
+### Create gatsby-node.js file
+
+Create a `gatsby-node.js` file at the root of your project if it doesn’t already exist, and add the code below to the `gatsby-node.js` file so your app can resolve paths without error:
```
Comment (shadcn) [inline apps/www/content/docs/installation/gatsby.mdx:57]: Same here. Remove _without error_.

**Review by shadcn — CHANGES_REQUESTED** [review]
This looks good. Left a few minor nits. Thanks for your help with this. Really appreciate it.

**Comment by moinulmoin** [comment]
> This looks good. Left a few minor nits. Thanks for your help with this. Really appreciate it.

Thanks, added those changes.

**Review by shadcn — APPROVED** [review]

**Pull request merged**